tests/php/test-amp-style-sanitizer.php

```diff
@@ -700,12 +700,19 @@ public function get_amp_selector_data() {
 				'.video{color:blue;} audio.audio{color:purple;} .iframe{color:black;} .img{color:purple;} .form:not(form){color:green;}',
 				'.video{color:blue}amp-audio.audio{color:purple}.iframe{color:black}.img{color:purple}.form:not(form){color:green}',
 			],
+			'focus_within_classes' => [
```

Comment:
Test case has both positive (replace `focus`) and negative (don't replace `focused`) cases. 👍